### PR TITLE
[Package Signing] Updated Test.Utility dependency

### PR DESCRIPTION
**Pull request by loic-sharma**

### CODE REVIEW

File: tests/Validation.PackageSigning.ValidateCertificate.Tests/Support/CertificateIntegrationTestFixture.cs [hunk 1]
```diff
@@ -86,11 +84,49 @@ void CustomizeAsSigningCertificate(X509V3CertificateGenerator generator)
...
+
+            void CustomizeAsSigningCertificate(X509V3CertificateGenerator generator)
+            {
+                generator.AddSigningEku();
+                generator.AddAuthorityInfoAccess(ca2, addOcsp: true, addCAIssuers: true);
+            }
+
+            var (publicCertificate, certificate) = IssueCertificate(ca2, "Revoked Signing", CustomizeAsSigningCertificate);
+
+            var caCert = new X509Certificate2();
```
Comment (dtivel) [inline tests/Validation.PackageSigning.ValidateCertificate.Tests/Support/CertificateIntegrationTestFixture.cs]: You can shorten this to:

    var caCert = new X509Certificate2(ca.Certificate.GetEncoded());

Comment (loic-sharma) [reply tests/Validation.PackageSigning.ValidateCertificate.Tests/Support/CertificateIntegrationTestFixture.cs]: Sounds good. I went ahead and made an extension method to clean this up even further.
